ModelManager.cpp

```diff
@@ -26,22 +26,22 @@ bool NnapiModelInfo::updateOutputshapes(size_t outputIndex, std::vector<size_t>&
 
 bool NnapiModelInfo::initializeRunTimeOperandInfo() {
     // initialize runtime operand info from model.
-    const size_t count = mModel.operands.size();
-    ALOGD("Operand size = %d\n", count);
+    const size_t count = mModel.main.operands.size();
+    ALOGD("Operand size = %zu\n", count);
     if (!count) {
         ALOGE("NNERR:Operand Count is 0");
         return false;
     }
     mOperands.resize(count);
-    mOutputShapes.resize(mModel.outputIndexes.size());
+    mOutputShapes.resize(mModel.main.outputIndexes.size());
 
     // Start by setting the runtime info to what's in the model.
-    for (size_t i = 0; i < count; i++) {
-        const Operand& from = mModel.operands[i];
-        dumpOperand(i, mModel);
+    for (int i = 0; i < count; i++) {
```

Comment:
this should be size_t else we will run into compiler warnings. Applies to all for loops modified in this PR